management/executive/src/executive.cc

```diff
@@ -3348,6 +3434,82 @@ bool Executive::Undock(ff_msgs::CommandStampedPtr const& cmd) {
   return docked;
 }
 
+bool Executive::UnloadNodelet(ff_msgs::CommandStampedPtr const& cmd) {
+  NODELET_INFO("Executive executing unload nodelet command!");
+  bool success = true;
+  std::string err_msg = "Unload nodelet parameters must be strings.";
+  ff_msgs::UnloadLoadNodelet unload_load_nodelet_srv;
+  unload_load_nodelet_srv.request.load = false;
```

Comment:
are these two functions just the same with this set to true or false? if so should combine them with a boolean argument

Reply:
Good point!

Reply:
I combined the code into one function and tried to make it cleaner.